src/AnnenErfaring/Seksjon.elm

```diff
@@ -513,13 +524,49 @@ update msg (Model model) =
                 _ ->
                     IkkeFerdig ( Model model, Cmd.none )
 
+        VilSlettePåbegynt ->
```

Comment:
Er det kanskje bedre om vi bruker ordet "Forkaste" i stedet for "Slette"? Er ikke noe problem nå, men jeg tenker at det kan bli forvirrende hvis vi senere legger til mulighet til å faktisk slette gamle ting fra CVen.